src/containers/JobseekerDashboard.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {Hidden, Link, Box, Grid, makeStyles, Typography } from '@material-ui/core';
+
+import FilterListIcon from '@material-ui/icons/FilterList';
+
+import { Navbar, Footer, PopUpComponent, StatsComponent, FilterComponent, SummaryComponent } from '../common';
+import oracle from '../assets/svg/oracle.svg';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import oracle from "../assets/svg/oracle.svg";
```